source/FreeRTOS_ND.c

```diff
@@ -479,14 +479,14 @@
         void FreeRTOS_PrintNDCache( void )
         {
             BaseType_t x, xCount = 0;
+            char pcBuffer[ 40 ];
```

Comment:
What does this change, i do not see any flag difference between the 2 locations.